RedisEnterprise-DNS-Migration/migration_steps.md

```diff
@@ -0,0 +1,82 @@
+# Migrate to the new Redis Enterprise Private DNS Zone
+
+This document is meant to help you migrate from an old form of the Private DNS Zone that is required to make private endpoints work with Redis Enterprise caches to a new form. The new form is a little simpler, better aligns with other services in Azure, and enables the automatic management of A records in the Private DNS Zone. Throughout this process, your infrastructure will continue to work as expected.
+
+## How to migrate to the new private DNS zone
+
+1. Identify Azure Virtual Networks (VNets) that have a private DNS zone
+    using the form:
+
+    ```{region}.privatelink.redisenterprise.cache.azure.net```
+
+2. For each of these VNets, create a new private DNS zone with the
+    name: privatelink.redisenterprise.cache.azure.net
```

Comment:
Just to double-check on Fran's earlier comment, this might be already as desired, or maybe a missed copy-paste to match the backtick formatting from step 1?

```suggestion
    name:
    
    ```privatelink.redisenterprise.cache.azure.net```
```